src/test/java/uk/gov/hmcts/reform/divorce/validationservice/rules/divorce/d8/D8PetitionerEmailTest.java

```diff
@@ -1,54 +1,54 @@
-package uk.gov.hmcts.reform.divorce.validationservice.rules.divorce.d8;
```

Comment:
Remove this class if this class is not necessary

Reply:
Done